framework/Optimizers/Optimizer.py

```diff
@@ -614,8 +614,9 @@ def initialize(self,externalSeeding=None,solutionExport=None):
     if self.solutionExport is None:
       self.raiseAnError(IOError,'The results of optimization cannot be obtained without a SolutionExport defined in the Step!')
 
-    if type(solutionExport).__name__ != "PointSet":
-      self.raiseAnError(IOError,'solutionExport type must be a PointSet. Got '+ type(solutionExport).__name__+ '!')
+    if type(solutionExport).__name__ not in ["PointSet","DataSet"]:
+      self.raiseAnError(IOError,'solutionExport type must be a PointSet or DataSet. Got '+\
```

Comment:
out of curiosity....if we can handle DataSet, why we can not handle HistorySet?

Reply:
By default the optimizer solution export contains scalars in the output, which disqualifies using a history set.